lib/node_modules/@stdlib/array/base/cuany-by/test/test.main.js

```diff
@@ -0,0 +1,261 @@
+
+	x = new Complex128Array( [ 1.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0 ] );
+	actual = cuanyBy( x, predicate );
+	expected = [ true, true, true, true, true ];
+	t.deepEqual( actual, expected, 'returns expected value' );
+
+	t.end();
+
+	function predicate( value ) {
+		return ( value > 0 );
+	}
+});
+
+tape( 'the function cumulatively tests whether at least one element in a provided array passes a test implemented by a predicate function (complex64)', function test( t ) {
+	var expected;
+	var actual;
+	var x;
+
+	x = new Complex64Array( [ 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0 ] );
+	actual = cuanyBy( x, predicate );
+	expected = [ false, false, true, true, true ];
+	t.deepEqual( actual, expected, 'returns expected value' );
+
+	x = new Complex64Array( [ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0 ] );
+	actual = cuanyBy( x, predicate );
+	expected = [ false, false, false, false, false ];
+	t.deepEqual( actual, expected, 'returns expected value' );
+
+	x = new Complex64Array( [ 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0 ] );
+	actual = cuanyBy( x, predicate );
+	expected = [ true, true, true, true, true ];
+	t.deepEqual( actual, expected, 'returns expected value' );
+
+	x = new Complex64Array( [ 0.0, 0.0, 1.0, 0.0, 0.0, 0.0 ] );
+	actual = cuanyBy( x, predicate );
+	expected = [ false, true, true ];
+	t.deepEqual( actual, expected, 'returns expected value' );
+
+	x = new Complex64Array( [ 1.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0 ] );
+	actual = cuanyBy( x, predicate );
+	expected = [ true, true, true, true, true ];
+	t.deepEqual( actual, expected, 'returns expected value' );
+
+	t.end();
+
+	function predicate( value ) {
+		return ( value > 0 );
+	}
+});
+
+tape( 'the function cumulatively tests whether at least one element of an accessor array is truthy (accessor)', function test( t ) {
+	var expected;
+	var actual;
+	var x;
+
+	x = toAccessorArray( [ false, false, true, false, false ] );
+
+	actual = cuanyBy( x, predicate );
+	expected = [ false, false, true, true, true ];
+	t.deepEqual( actual, expected, 'returns expected value' );
+
+	x = toAccessorArray( [ false, false, false, false, false ] );
+	actual = cuanyBy( x, predicate );
+	expected = [ false, false, false, false, false ];
+	t.deepEqual( actual, expected, 'returns expected value' );
+
+	x = toAccessorArray( [ true, true, true, true, true ] );
+	actual = cuanyBy( x, predicate );
+	expected = [ true, true, true, true, true ];
+	t.deepEqual( actual, expected, 'returns expected value' );
+
+	x = toAccessorArray( [ false, true, false ] );
+	actual = cuanyBy( x, predicate );
+	expected = [ false, true, true ];
+	t.deepEqual( actual, expected, 'returns expected value' );
+
+	x = toAccessorArray( [ true, false, false, false, false ] );
+	actual = cuanyBy( x, predicate );
+	expected = [ true, true, true, true, true ];
+	t.deepEqual( actual, expected, 'returns expected value' );
+
+	t.end();
+
+	function predicate( value ) {
+		return ( value > 0 );
```

Comment:
Would change as-follows given that the tested values are Booleans:

```suggestion
		return value;
```